telemetry/README.md

```diff
@@ -0,0 +1,67 @@
+# VIP Telemetry Library
+
+## Tracks
+
+Tracks is an event tracking tool used to understand user behaviour within Automattic. This library provides a way for plugins to interact with the Tracks system and start recording events.
+
+### How to use
+
+Example:
+
+```php
+use Automattic\VIP\Telemetry\Tracks
+
+function track_post_status( $new_status, $old_status, $post ) {
+	$tracks = new Tracks( 'myplugin_' );
+
+	$tracks->record_event( 'post_status_changed', [
+		'new_status' => $new_status,
+		'old_status' => $old_status,
+		'post'       => (array) $post,
```

Comment:
Love this easy example. A whole `$post` [may be overkill](https://github.com/Automattic/vip-workflow-plugin/pull/21#discussion_r1733516494) for something we'd want to recommend sending to telemetry. How about:

```suggestion
		'post_id'       => $post->ID,
```

or similar instead?